### DISCUSSION TITLE
Set order of conditional processgroup naming rules

### DISCUSSION THREAD

**Discussion by mdicss** [Dynatrace/dynatrace-configuration-as-code#315] [Archive] [answered]
Hi
In the dynatrace settings, you can change the order of the process group custom naming rules. With monaco, I get a random order of the rules. It should be possible to set the order of the rules.
Probably this is missing in the dynatrace api, so it has to be implemented there first?

**Comment by tatomir146** [comment, marked as answer]
Hi @mdicss, this will be implemented once monaco supports settings 2.0. Settings 2.0 API will also allow versioning. We will implement this after monaco 2.0 is released.
